### PR TITLE
Send email when account goes from inactive to active

### PR DESCRIPTION
**Pull request by c0mster**
@DanielJMaher 
This should be ready to go.

Note there is a new config Key under LOCAL_DEVELOPMENT called TOEMAIL

This is used to set a email:to when testing emails on local.
If you have a local debug please add this key there

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@oceanzus Jim, check this for me?

File: ooiservices/app/main/user.py [hunk 1]
```diff
@@ -36,11 +37,31 @@ def put_user(id):
...
+def send_activate_email(to):
+    sender="help@ooi.rutgers.edu"
+    subject="OOI Registration Complete"
+    message=string.join(("From: %s" % sender,
+                         "To: %s" % current_app.config["TOEMAIL"] if current_app.config["ENVIORNMENT"] else to,
+                         "Subject: %s" % subject,
+                         "Your registration to the Ocean Observatories Initiative WebPortal has been completed.",
+                         "",
+                         "",
+                         "Please visit http://casadev.ooi.rutgers.edu to login to the system"),"\r\n")
```
Comment (DanielJMaher) [inline ooiservices/app/main/user.py]: Cam, Sorry for this.  Can you make a config variable SERVER_DNS in the config.yml with the defauly value "localhost", then change the code to read:

"Please visit http://%s to login to the system"),"\r\n" %current_app.config["SERVER_DNS"])

If you don't have time, let me know and I'll handle it

**Comment by birdage** [comment]
@DanielJMaher @FBRTMaka @oceanzus this looks good to me after the changes are made, merge when ready. requires testing

**Comment by c0mster** [comment]
@DanielJMaher 
Ready to test now

**Pull request merged**